backend/img_upload.ts

```diff
@@ -0,0 +1,45 @@
+import dotenv from 'dotenv'
+import axios from 'axios'
+import FormData from 'form-data'
+import fs from 'fs'
+
+// initialize configuration
+dotenv.config()
+
+const PINATA_JWT = process.env.PINATA_JWT ?? ''
+const PINATA_PINFILE_URL = process.env.PINATA_PINFILE_URL ?? ''
+
+const JWT = `Bearer ${PINATA_JWT}`
+
+const pinFileToIPFS = async (name: string) => {
+  const formData = new FormData()
+  const IMG_PATH = './backend/me.jpg'
+
+  const file = fs.createReadStream(IMG_PATH)
+  formData.append('file', file)
+
+  const metadata = JSON.stringify({
+    name: name,
+  })
+  formData.append('pinataMetadata', metadata)
+
+  const options = JSON.stringify({
+    cidVersion: 0,
+  })
+  formData.append('pinataOptions', options)
+
```

Comment:
Add some docstrings.
-- Explain how you're uploading an image and and metadata **on Pinata** BEFORE one can actually update the NFTs metadata on Solana MetapleX.